src/test/java/com/commercetools/emailprocessor/model/StatisticsTest.java

```diff
@@ -43,7 +42,6 @@ public void ofError_isCalledForTenant_shouldShowCorrectStatstic() throws Excepti
         assertEquals(statistics.getPermanentErrors(), 0);
         assertEquals(statistics.getTemporaryErrors(), 0);
         assertEquals(statistics.getProcessed(), 0);
-        assertEquals(statistics.getNotProcessed(), 0);
```

Comment:
Why are all these asserts removed in this unit test?

Reply:
because the attribute "notprocessed" doesn`t exist anymore in the statistic, because this state could not be "reached" in the current implementation

Reply:
Why is this not possible? can you please clarify this change in logic as it's not described any where in the PR description?

Reply:
It is described in the pr-description:

The check, if emailobects are in state "pending" , is directly made within the "CustomObjectQuery" and not while iteration over the returned emailobjects.

So the Result of the customobject query doesn`t has any object which shoudn´t be  processed

Reply:
ok